nemo_skills/evaluation/metrics/translation_metrics.py

```
@@ -0,0 +1,80 @@
+            src_lang, tgt_lang = key.split("_")
+            preds = self.translation_dict[key]["preds"]
+            gts = self.translation_dict[key]["gts"]
+
+            tokenize = "13a"
+            tgt_lang = key.split("_")[-1]
+            if tgt_lang == "ja":
+                tokenize = "ja-mecab"
+            
+            bleu_score = corpus_bleu(preds, [gts], tokenize=tokenize).score
+            metrics_dict[key] = {"bleu": bleu_score}
+            self.aggregation_dict[f"xx_xx"].append(bleu_score)
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)